Jack edits about seels and text

Nashet opened the pull request.

Nashet commented:
So I added @historyjackz your changes to master, thanks!
I made it a little bit in a weird way but I'm not experienced with pull requests.
About top panel text - It would be changed a little in 18 version - education % will be added, and something else. I'm not sure about idea to use abbreviation like ENG.

historyjackz commented:
Right. This is a learning experience for us both! 
I understand why you put certain things such as "Money" or "You Rule" and perhaps because I am used to playing games like this, which I assume is your audience(?) I don't think those are necessary once established. For example using a monetary symbol versus the word Money. The $ is a universal symbol for the USD which translates to money or we could even use the "¤" symbol which, according to Wikipedia is the generic symbol for money. If the monetary symbol is explained in the opening panel, this would give us the ability to make a cleaner UI. 
(https://en.wikipedia.org/wiki/Currency_symbol)
I also believe Science Points sounds clunky. There are certainly other abbreviations such as Tech Points (Technology Points) or even Age Points (Because the inventions really belong to different ages). 

Possibly Tech inventions (Steam power, etc), and Ideas/Theories (such as capitalism/laissez-faire or collectivism) could be separated into different categories with respective points. This would mirror Vic/HOI4/EU4.  I understand everything is still being worked on, but it could be put into a future implementation.

There is also the possibility of making the giant buttons for Military, Inventions, etc. Either shorter by abbreviating to three letters or even one letter to give more room on the Top Bar. The abbreviations would not hinder gameplay, because again just as with the currency symbols we can explain the abbreviation within the Wiki and the Startup menu.

Nashet commented:
>"You Rule"

The thing is that many players don't understand what they doing. For example, huge amount of players wasn't able to press full screen button, so I added that green blinking memo. Several guys didn't know how to push Play button. I'd like to add big blinking arrows saying YOU PLAY THAT COUNTRY, YES THAT COUNTRY if wouldn't too  complicated. Even I'm myself sometimes confused about what country I play now.

>"Money" 

In 0.18.0 version I converted almost all money variables to Money class which provides common postfix (currently - "Gold" since money now are actual gold digged from pits).
But there is problem with visualization integer numbers. Money now shown as 12,345.123 Gold. Thats standard number format ( for USA?). For me it looks ugly. May be I should change it to 12 345.123 Gold or even to 12K Gold
![18finance](https://user-images.githubusercontent.com/5767687/36535285-327cf08e-17da-11e8-96a3-1d1d1af2cfc6.png)

>I also believe Science Points sounds clunky. There are certainly other abbreviations such as Tech Points (Technology Points)

Technology Points sounds great.

>Possibly Tech inventions (Steam power, etc), and Ideas/Theories (such as capitalism/laissez-faire or collectivism) could be separated into different categories

Added ticket #426 but that's low priority 

> The abbreviations would not hinder gameplay

Regular players (well, some of them) can't even hit play button. And, there is enough space in top panel, isn't it? I tested it on different screen sizes, 1280x1024 may have issues but it's rare resolution.

By the way, there is some guy trying to improve UI sprites
![15191616763600](https://user-images.githubusercontent.com/5767687/36535636-67c288fc-17db-11e8-8b93-40c34e6e995c.png)

historyjackz commented:
I don't think we should discredit players. If we make it obvious the country they are playing by either making a tutorial screen or color coding players the same color always? Since you have the "Focus on country" zoom function already included it helps to identify the players country. 


Again, I think anyone can learn the game easily if we simplify controls and the menu. I've made two ideas of how I think it could be implemented. 
![Drawing.jpeg](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a8658281112da001280bbff/125516c66e82c728ace21e0d46cc869b348b8daecda5fd53b652fa912a5265296b51fa7b2180e77eeb1b22535e0e47ac415b5d10f8ffdd28a6a42062c80d55b98e30063e937f03a6aa8317b500bf21824c87b7b58f48c0ee3e37aa.jpeg)
By nesting some of the buttons we can simplify and control what people see. I know this isn't an urgent issue, but its something to consider for the future while coding it. (My art skills aren't that great and I did this in a couple minutes.) 

For the monetary issue. I, as an American, prefer the comma within the money, but it could be beneficial to display 9,999 and at 10,000 start displaying it as 10K. 
Using Gold is probably the best method for currency though. At least until a more advanced system could be designed.

historyjackz commented:
In my UI Edit branch I've migrated the Politics and population buttons within a Government button! It worked pretty well. The Government window also displays the "You Rule: COUNTRY" at the top.

Nashet commented:
>Again, I think anyone can learn the game easily if we simplify controls and the menu. I've made two ideas of how I think it could be implemented.

That are good ideas, cause they say that people can only take in mind about 7 items at once, but how do you see grouping several buttons at once, like finance, enterprises, trade?

historyjackz commented:
This is just a quick sketch up I did of a potential GUI setup. 
Each panel would be setup with a series of tabs, which would display within the respective menu, rather than open a bunch of new panels. 
Or at least present an overview and then open new panels. 

![Drawing-2.jpeg](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a8658281112da001280bbff/125516c66e82c728ace21e0d46cc869b348b8daec2bbfd0fe64de29b234324676b5ee17e29d5b22dbb142f58565d4ffd48585f47fead8d2df2b53c62964e49bc8f3900299f7744eeaa880dbe06b0218e4e8cb2bd8d4ac6e03f24e699c2.jpeg)
Obviously these aren't sized to how they could be.

Nashet commented:
Sorry for late answer,  @historyjackz. I got you idea of tabs. I like it, cause it structurises info, yet its a big work to do these tabs. Also, I use sometimes several windows at once for debug reason..

I have several bugs in priority now.. I and I want to release 18 version soon. There would be population change map, btw. https://twitter.com/Nashet18/status/968247309921382401
